### PR TITLE
Bump default k8s version to 1.18

### PR DESCRIPTION
**Pull request by machacekondra**
And also use integrated CNAO, instead of explicit installation.

Signed-off-by: Ondra Machacek <omachace@redhat.com>

### CODE REVIEW

File: cluster/common.sh [hunk 1]
```diff
@@ -2,15 +2,6 @@
 
 set -e
 
-function install_cnao {
```
Comment (pkliczewski) [inline cluster/common.sh:5]: Why we do not need this anymore?

Comment (machacekondra) [reply cluster/common.sh:5]: It's part of the `kubevirtci` repo [now](https://github.com/kubevirt/kubevirtci/tree/master/cluster-up/cluster/k8s-1.17#bringing-the-cluster-up-with-cluster-network-addons-operator-provisioned). It's enabled just by passing `KUBEVIRT_WITH_CNAO=true`

Comment (machacekondra) [reply cluster/common.sh:5]: To make it effect, this must be merged before: https://github.com/kubevirt/project-infra/pull/449

Comment (pkliczewski) [reply cluster/common.sh:5]: OK, thanks

**Review by pkliczewski — APPROVED** [review]

**Review by jakub-dzon — APPROVED** [review]

**Pull request merged**